mccsearch/code/mainProg.py

```diff
@@ -55,35 +55,35 @@ def main():
     # -------------------------------------------------------------------------------------------------
 
     # let's go!
-    print "\n -------------- Read MERG Data ----------"
+    print("\n -------------- Read MERG Data ----------")
     mergImgs, timeList = mccSearch.readMergData(CEoriDirName)
-    print ("-" * 80)
+    print(("-" * 80))
```

Comment:
Recommendation to change `print(("-" * 80))` to `print("-" * 80)` in eight places.  The double nested parens `(( ))` are not needed and only slow down your reader.  Repeated on lines: 60, 71, 73, 76, 78, 81, 84, 107.